src/shared/components/results/__tests__/__snapshots__/ResultsData.spec.tsx.snap

```diff
@@ -3968,7 +3968,11 @@ exports[`ResultsData component should render the table view 1`] = `
           <th
             class=""
           >
-            Entry
+            <span
+              aria-expanded="false"
+            >
```

Comment:
Is this an artifact of the tooltip library?

Reply:
It looks like it - https://github.com/tvkhoa/react-tippy#accessibility